src/shared/components/headerClipMaskLeft/headerClipMaskLeft.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import styles from './headerClipMaskLeft.css';
+
+class HeaderClipMaskLeft extends Component {
```

Comment:
This component and the `HeaderClipMaskRight` component will give us the clipping appearance we were seeing in the mockup. Not a big fan of this solution, but I think that if a browser doesn't support SVG, then it will simply not render it. Open to suggestions, but if it's too much of a hassle, I'll probably just remove it.

Reply:
I actually like, but I'd go a step further and join Left and Right to just make `HeaderClipMask` for an import.